app/models/metric/capture.rb

```diff
@@ -43,12 +43,14 @@ def self.alert_capture_threshold(target)
                 ::Settings.performance.capture_threshold_with_alerts.default)
   end
 
-  def self.perf_capture_timer(zone = nil)
+  def self.perf_capture_timer(ems_id = nil)
     _log.info("Queueing performance capture...")
+    return unless ems_id
```

Comment:
make ems_id manditory and have zone messages blow up

Customer should empty the queue between upgrades anyway.
There will only be 1 entry per ems - so minimal noise in logs.

/via @agrare